templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -204,1410 +204,18 @@
         }
       EOF
 
-      # This is a bit ridiculous, but requiring people to install Go in order to build is slightly
-      # more ridiculous IMO.
-      # TODO(jcanizales): Translate err_data_generate.go into a Bash or Ruby script.
+      # To build boringssl, we need the generated file err_data.c, which is normally generated
```

Comment:
Add a note in the front to notify people that they should run `generate_projects.sh` twice to get the gererated `BoringSSL-GRPC.podspec` correct, in case people get confused about why Sanity Checker fails even after running `generate_projects.sh`.